### PR TITLE
Enable cluster topology for OpenStack by default and update cluster-openstack default version to 0.3.0

### PR DESCRIPTION
**Pull request by tfussell**
The latest version of cluster-openstack is 0.3.0.

In the next `kubectl-gs` release we will get rid of the old non-cluster topology cluster templating entirely. During the transition it should be the default.

### CODE REVIEW

**Review by anvddriesch — APPROVED** [review]

**Review by rossf7 — APPROVED** [review]

File: cmd/template/cluster/flag.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func (f *flag) Init(cmd *cobra.Command) {
 
 	// OpenStack App only.
 	cmd.Flags().StringVar(&f.ClusterApp.ClusterUserConfigMap, flagClusterUserConfigMap, "", "Path to the user values configmap YAML file for Cluster App (OpenStack App CR only).")
-	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.1.0", "Cluster App version to be installed. (OpenStack App CR only).")
+	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.2.1", "Cluster App version to be installed. (OpenStack App CR only).")
```
Comment (kopiczko) [inline cmd/template/cluster/flag.go:133]:
```suggestion
	cmd.Flags().StringVar(&f.ClusterApp.ClusterAppVersion, flagClusterAppVersion, "0.2.2", "Cluster App version to be installed. (OpenStack App CR only).")
```

Comment (marians) [reply cmd/template/cluster/flag.go:133]: If 0.2.2 is the right version, please update the PR title and description.

Comment (tfussell) [reply cmd/template/cluster/flag.go:133]: Thanks, yes, I was holding off as I was still deciding what the correct default version should be. I will update the title and description if I change it.

**Comment by tfussell** [comment]
Made another flag change and updated the version to match with the latest release.

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**